### PR TITLE
Run `make gen`

### PR DESCRIPTION
**Pull request by ericvn**
**Please provide a description of this PR:**

Run make gen with new build-tools

### CODE REVIEW

File: operator/pkg/patch/patch.go [hunk 1]
```diff
@@ -15,78 +15,81 @@
...
+  - [key:value] selects a list entry in list c which contains an entry with key:value
+  - [list_entry_value] selects a list entry in list d which is a regex match of list_entry_value.
 
 Some examples are given below. Given a resource:
 
 kind: Deployment
 metadata:
-  name: istio-citadel
-  namespace: istio-system
+
```
Comment (howardjohn) [inline operator/pkg/patch/patch.go:25]: this is not right... shouldn't have a space like this

File: pkg/test/framework/components/istio/kube.go [hunk 2]
```diff
@@ -582,7 +582,7 @@ func waitForIstioReady(ctx resource.Context, c cluster.Cluster, cfg Config) erro
 	return nil
 }
 
-// configureDirectAPIServiceAccessBetweenClusters - create a remote secret of cluster `c`` and place
+// configureDirectAPIServiceAccessBetweenClusters - create a remote secret of cluster `c“ and place
```
Comment (howardjohn) [inline pkg/test/framework/components/istio/kube.go:585]: this doesn't look right

File: tools/bug-report/pkg/config/config.go [hunk 3]
```diff
@@ -50,28 +50,29 @@ const (
...
 // Examples:
 //
-// 1. All pods in test-namespace with label "test=foo" but without label "private" (with any value):
-//      include:
-//        test-namespace/*/*/test=foo
-//      exclude:
-//        test-namespace/*/*/private
+//  1. All pods in test-namespace with label "test=foo" but without label "private" (with any value):
+//     include:
+//     test-namespace/*/*/test=foo
```
Comment (howardjohn) [inline tools/bug-report/pkg/config/config.go:55]: this lost indent

File: tools/bug-report/pkg/config/config.go [hunk 4]
```diff
@@ -50,28 +50,29 @@ const (
...
 //
-// 4. Container logs for all containers called "istio-proxy":
-//      include:
-//        */*/*/*/*/istio-proxy
+//		include:
+//		  */*/*prometheus*
+//		exclude:
+//		  */*/*prometheus*/*/internal=true
 //
+//	 4. Container logs for all containers called "istio-proxy":
```
Comment (ericvn) [inline tools/bug-report/pkg/config/config.go:73]: I notice 4. is indented. Doesn't seem correct.

File: cni/pkg/repair/repair_test.go [hunk 5]
```diff
@@ -237,8 +237,9 @@ func TestBrokenPodReconciler_detectPod(t *testing.T) {
 
 // Test the ListBrokenPods function
 // TODO:(stewartbutler) Add some simple field selector filter test logic to the client-go
-//  fake client. The fake client does NOT support filtering by field selector,
-//  so we need to add that ourselves to complete the test.
+//
```
Comment (howardjohn) [inline cni/pkg/repair/repair_test.go:240]: no need for space

File: operator/pkg/patch/patch.go (hunk 1 above)
Comment (howardjohn) [inline operator/pkg/patch/patch.go:25]: should indent the whole thing so its 'code' and we dont get these spaces

File: pkg/dns/client/dns_test.go [hunk 6]
```diff
@@ -323,12 +323,16 @@ func testDNS(t *testing.T, d *LocalDNSServer) {
 }
 
 // Baseline:
-//      ~150us via agent if cached for A/AAAA
-//      ~300us via agent when doing the cname redirect
-//      5-6ms to upstream resolver directly
-//      6-7ms via agent to upstream resolver (cache miss)
+//
+//	~150us via agent if cached for A/AAAA
```
Comment (howardjohn) [inline pkg/dns/client/dns_test.go:327]: nit: should probably be list instead of code font

File: pkg/istio-agent/health/health_probers.go [hunk 7]
```diff
@@ -88,7 +88,8 @@ func NewHTTPProber(cfg *v1alpha3.HTTPHealthCheckConfig, ipv6 bool) *HTTPProber {
 }
 
 // HttpProber_Probe will return whether or not the target is healthy (true -> healthy)
-// 	by making an HTTP Get response.
+//
```
Comment (howardjohn) [inline pkg/istio-agent/health/health_probers.go:91]: nit: no need for space

File: pkg/kube/inject/initializer.go [hunk 8]
```diff
@@ -31,9 +31,12 @@ import (
 // "kube-system": The namespace for objects created by the Kubernetes system.
 // "kube-public": This namespace is mostly reserved for cluster usage.
 // "kube-node-lease": This namespace for the lease objects associated with each node
-//    which improves the performance of the node heartbeats as the cluster scales.
+//
```
Comment (howardjohn) [inline pkg/kube/inject/initializer.go:34]: remove extra space

File: pkg/test/framework/components/echo/echotest/filters.go [hunk 9]
```diff
@@ -93,9 +97,11 @@ func (t *T) applyCombinationFilters(from echo.Instance, to echo.Instances) echo.
 // other "regular" pods that aren't part of the same Service. Pods that are part of the same Service but are in a
 // different cluster or revision will still be included.
 // Example:
-//     The full set of apps is a, b, c, headless, naked, and vm.
-//     The plain-pods are a, b and c.
-//     This filter would result in a, headless, naked and vm.
+//
+//	The full set of apps is a, b, c, headless, naked, and vm.
```
Comment (howardjohn) [inline pkg/test/framework/components/echo/echotest/filters.go:101]: nit: list?

File: pkg/test/framework/components/echo/echotest/run.go [hunk 10]
```diff
@@ -122,11 +122,11 @@ func (t *T) fromEachCluster(ctx framework.TestContext, testFn perClusterTest) {
 // may appear as a target in multiple subtests.
 //
 // Example: Given a as the only source, with a, b, c, d as destinationsand n = 3, we get the following subtests:
-//     a/to_a_b_c/from_cluster_1:
-//     a/to_a_b_c/from_cluster_2:
-//     a/to_b_c_d/from_cluster_1:
-//     a/to_b_c_d/from_cluster_2:
 //
+//	a/to_a_b_c/from_cluster_1:
```
Comment (howardjohn) [inline pkg/test/framework/components/echo/echotest/run.go:126]: nit: list? Maybe list of code (not sure you can do that)

File: pkg/test/framework/components/echo/echotest/setup.go [hunk 11]
```diff
@@ -30,13 +30,14 @@ type (
...
 // these tests would all run before the context is cleaned up:
-//     a/to_b/from_cluster-1
-//     a/to_b/from_cluster-2
-//     a/to_c/from_cluster-1
-//     a/to_c/from_cluster-2
-//     cleanup...
-//     b/to_a/from_cluster-1
-//     ...
+//
+//	a/to_b/from_cluster-1
```
Comment (howardjohn) [inline pkg/test/framework/components/echo/echotest/setup.go:34]: same comment about list of code

File: tools/bug-report/pkg/config/config.go (hunk 4 above)
Comment (howardjohn) [reply tools/bug-report/pkg/config/config.go:73]: yeah it renders wrong

**Review by howardjohn — APPROVED** [review]

**Review by therealmitchconnors — APPROVED** [review]

**Comment by ericvn** [comment]
Will also need this plus successive PRs cherry-picked to move to Go 1.19 in the release-1.15 branch.

**Pull request merged**